Add repo fails on windows

I am using the local binary

```
<useLocalHelmBinary>true</useLocalHelmBinary>
```
```
[ERROR] Failed to execute goal com.kiwigrid:helm-maven-plugin:4.8:init (package) on project hello-jaxrs: Error processing command
 [C:\projects\hello-jaxrs\target/helm\helm repo add helm-repo https://************/repository/helm/ --username=************--password=************]: 
Cannot run program "C:\projects\hello-jaxrs\target/helm\helm": 
CreateProcess error=2, The system cannot find the file specified -> [Help 1]
```

--- a/src/main/java/com/kiwigrid/helm/maven/plugin/AbstractHelmMojo.java
+++ b/src/main/java/com/kiwigrid/helm/maven/plugin/AbstractHelmMojo.java
@@ -97,12 +97,12 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 	Path getHelmExecuteablePath() throws MojoExecutionException {
 		String helmExecutable = SystemUtils.IS_OS_WINDOWS ? "helm.exe" : "helm";
 		Optional<Path> path;
-		if(isUseLocalHelmBinary() && isAutoDetectLocalHelmBinary()){
+		if (isUseLocalHelmBinary() && isAutoDetectLocalHelmBinary()) {
 			path = findInPath(helmExecutable);
-		}else {
+		} else {
 			path = Optional.of(Paths.get(helmExecutableDirectory, helmExecutable))
-							.map(Path::toAbsolutePath)
-							.filter(Files::exists);
+					.map(Path::toAbsolutePath)
+					.filter(Files::exists);
 		}
 
 		return path.orElseThrow(() -> new MojoExecutionException("Helm executable is not found."));
@@ -110,6 +110,7 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 
 	/**
 	 * Finds the absolute path to a given {@code executable} in {@code PATH} environment variable.
+	 *
 	 * @param executable the name of the executable to search for
 	 * @return the absolute path to the executable if found, otherwise an empty optional.
 	 */
@@ -117,11 +118,11 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 
 		final String[] paths = getPathsFromEnvironmentVariables();
 		return Stream.of(paths)
-					 .map(Paths::get)
-					 .map(path -> path.resolve(executable))
-					 .filter(Files::exists)
-					 .map(Path::toAbsolutePath)
-					 .findFirst();
+				.map(Paths::get)
+				.map(path -> path.resolve(executable))
+				.filter(Files::exists)
+				.map(Path::toAbsolutePath)
+				.findFirst();
 	}
 
 	String[] getPathsFromEnvironmentVariables() {
@@ -179,7 +180,7 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 	List<String> getChartDirectories(String path) throws MojoExecutionException {
 		List<String> exclusions = new ArrayList<>();
 
-		if(getExcludes() != null) {
+		if (getExcludes() != null) {
 			exclusions.addAll(Arrays.asList(getExcludes()));
 		}
 
@@ -418,7 +419,7 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 		return autoDetectLocalHelmBinary;
 	}
 
-	public void setAutoDetectLocalHelmBinary(final boolean autoDetectLocalHelmBinary) { 
-		this.autoDetectLocalHelmBinary = autoDetectLocalHelmBinary; 
+	public void setAutoDetectLocalHelmBinary(final boolean autoDetectLocalHelmBinary) {
+		this.autoDetectLocalHelmBinary = autoDetectLocalHelmBinary;
 	}
 }

--- a/src/main/java/com/kiwigrid/helm/maven/plugin/InitMojo.java
+++ b/src/main/java/com/kiwigrid/helm/maven/plugin/InitMojo.java
@@ -56,7 +56,7 @@ public class InitMojo extends AbstractHelmMojo {
 
 		if(isUseLocalHelmBinary()) {
 			verifyLocalHelmBinary();
-			getLog().info("Using local HELM binary ["+ getHelmExecutableDirectory() +"]");
+			getLog().info("Using local HELM binary ["+ getHelmExecuteablePath() +"]");
 		} else {
 			downloadAndUnpackHelm();
 		}
@@ -65,9 +65,8 @@ public class InitMojo extends AbstractHelmMojo {
 			for (HelmRepository repository : getHelmExtraRepos()) {
 				getLog().info("Adding repo " + repository);
 				PasswordAuthentication auth = getAuthentication(repository);
-				callCli(getHelmExecutableDirectory()
-								+ File.separator
-								+ "helm repo add "
+				callCli(getHelmExecuteablePath()
+								+ " repo add "
 								+ repository.getName()
 								+ " "
 								+ repository.getUrl()

